src/GraphControl/Control/Grapher.cpp

```diff
@@ -784,6 +792,7 @@ namespace GraphControl
 
                 if (needsRenderPass)
                 {
+                    IsKeepCurrentView = true;
```

Comment:
IsKeepCurrentView should change in _response_ to the GraphViewChangedEvent. It doesn't make sense for this property to be changed from this function. GraphViewChanged can be fired from more than one location. Clicking the +/- buttons do not change IsKeepCurrentView currently, as an example.